charts/acp/templates/configmap.yaml

```diff
@@ -27,7 +27,7 @@ data:
       {{- with .Values.sql }}
         {{- toYaml . | nindent 6 }}
       {{- else }}
-      url: "postgres://root@{{ .Release.Name }}-cockroachdb:26257/defaultdb?sslmode=disable"
+      url: "postgres://root@{{ .Release.Name }}-cockroachdb-public:26257/defaultdb?sslmode=disable"
```

Comment:
Could we make it more configurable (username, db-name, port, SSL mode)?

Reply:
No, it is hardcoded for umbrella chart to work